Python/Product/Profiling/Profiling/CustomPythonInterpreterView.cs

```diff
@@ -0,0 +1,28 @@
+// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+namespace Microsoft.PythonTools.Profiling {
+    class CustomPythonInterpreterView : PythonInterpreterView {
+        public CustomPythonInterpreterView() :
+            base(Strings.LaunchProfiling_OtherInterpreter, "", null) {
+        }
+        public CustomPythonInterpreterView(string path) :
```

Comment:
Nit: you can collapse these two ctors into a single one with a default value: `string path = null`.